xml/http_cookies.xml

```diff
@@ -407,6 +407,15 @@
     <param pos="0" name="service.cpe23" value="cpe:/a:redmine:redmine:-"/>
   </fingerprint>
 
+  <fingerprint pattern="^(syracuse.sid.\d+)=.*">
```

Comment:
No need for the trailing `.*`

Reply:
Yep, I was just doing this to match the other signatures I was seeing. I'll remove it 👍